spec/rails_helper.rb

```diff
@@ -12,11 +15,48 @@ module Features
   include Formulaic::Dsl
 end
 
+VCR.configure do |c|
+  c.cassette_library_dir = 'spec/cassettes'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.